src/client/client4.js

```diff
@@ -891,27 +1052,39 @@ export default class Client4 {
 
     // Post Routes
     createPost = async (post) => {
+        this.trackEvent('api', 'api_posts_create', {channel_id: post.channel_id});
+
+        if (post.root_id != null && post.root_id !== '') {
+            this.trackEvent('api', 'api_posts_replied', {channel_id: post.channel_id});
```

Comment:
it seems that we are tracking both events, is that ok? well I think you are just bringing this from what is currently in the webapp